File: mobile/flutter_application_55555/lib/features/library/domain/entities/library_item.dart
class LibraryItem {
  final String id;
  final String title;
  final String content;
  final String imageUrl;
  final String sources;
  final String? shortDescription;
  final String? categoryId;
  final String? categoryName;

  LibraryItem({
    required this.id,
    required this.title,
    required this.content,
    required this.imageUrl,
    this.sources = '',
    this.shortDescription,
    this.categoryId,
    this.categoryName,
  });

  get createdAt => null;

  get updatedAt => null;
}

File: backend/DOGD/DOGD_API/Models/LibraryItem.cs
namespace DOGD_API.Models
{
    public class LibraryItem
    {
        public Guid Id { get; set; }
        public Guid CategoryId { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string? ShortDescription { get; set; }  
        public string ImageUrl { get; set; }
        public string? Sources { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

        public LibraryCategory Category { get; set; }
    }

}
